content/blog/gain-platform-superpowers-with-the-kebap-stack/index.md

```diff
@@ -0,0 +1,295 @@
+
+    This helps reduce the time required to establish development and production environments from months to minutes. #GitOps #Kubernetes #ArgoCD #Pulumi #Backstage
+---
+
+<img src="cncf.png" width="500px">
+
+In today's rapidly evolving tech ecosystem, it's crucial for any tool to seamlessly integrate with the Cloud Native
+Computing Foundation (CNCF) landscape. This vast array of technologies can seem overwhelming, but it presents immense
+opportunities for innovation and efficiency in software delivery.
+
+At Pulumi's Customer Experience Team, we specialize in guiding software infrastructure teams through their journey from
+traditional delivery paradigms to Cloud Native approaches. We've observed that the adoption of an infrastructure as code
+tool often serves as a catalyst for organizations to reassess their entire software delivery process. This reassessment
+frequently leads to the implementation of more streamlined, automated, and secure methodologies.
+
+While the CNCF landscape continues to expand with new tools emerging daily, we've identified recurring patterns in
+successful Cloud Native transformations. Pulumi, as a versatile infrastructure as code platform, plays a pivotal role in
+these patterns. It acts as a unifying force, enabling teams to leverage various CNCF technologies cohesively and
+efficiently.
+
+By embracing Pulumi and the wider CNCF ecosystem, organizations can create robust, scalable, and future-proof
+infrastructure solutions. This approach not only enhances operational efficiency but also positions teams to readily
+adopt and benefit from emerging Cloud Native technologies.
+
+## One Reference Architecture: The `KEBAP` Stack
+
+The `KEBAP` stack consists of the following components:
+
+- **K**ubernetes (with Kyverno as a bonus)
+- **E**xternal Secrets Operator
+- **B**ackstage
+- **A**rgo CD
+- **P**ulumi
+
+This combination forms a standardized GitOps stack applicable to various infrastructure projects. By implementing the
+`KEBAP` stack, we've consistently reduced the time required for teams to establish development and production
+environments from months to minutes.
+
+To following along, we are going to use
+the [capabilities of platforms](https://tag-app-delivery.cncf.io/whitepapers/platforms/#capabilities-of-platforms)
+definition from the CNCF working group on Platforms.
+
+<img src="base.png" width="500px">
+
+## Let's Break It Down
+
+Let's take a closer look at each component of the `KEBAP` stack and understand how they work together to streamline the
+software delivery process.
+
+The five layers of the `KEBAP` stack are designed to address different aspects of the software delivery process:
+
+### Kubernetes
+
+<img src="k.png" width="500px">
+
+Kubernetes serves as the orchestration layer. It's now the dominant container orchestration platform across the
```

Comment:
I think we are probably overloading definitions. Orchestration and infrastructure. Because Pulumi also does some orchestration on infrastructure as well. So we need to define the layers involved here. 

What layer of infra is K8s involved in and what layer of infra is Pulumi involved in? I think we need to clarify that so it makes more sense